packages/core/docs/general/where-to-start.md

```diff
@@ -0,0 +1,106 @@
+# Where to start?
+
+This document will show where you should start if you are:
+
+[[toc]]
+
+## Considering using Vue Storefront
+
+Vue Storefront is a **platform-agnostic e-commerce PWA frontend framework** that can work with any eCommerce backend API. Sounds great, right? But that is not all.
+
+With our product, you can easily solve the most common problems in the eCommerce e.g.
+- [Long time to market](/#long-time-to-market)
+- [Slow, unresponsive online shop](/#slow-unresponsive-online-shop)
+- [Unwieldy architectural decisions](/#unwieldy-architectural-decisions)
+- [Painful or impossible migrations](/#painful-or-impossible-migrations)
+- [Lack of platform-specific competencies](/#lack-of-platform-specific-competencies)
+- [Lack of flexibility](/#lack-of-flexibility)
+
+Also, we are proud of:
+- 300+ [Live Implementations](https://www.vuestorefront.io/live-projects)
+- 100+ [Agency Partners](https://www.vuestorefront.io/partner-agencies)
+- 4500+ [Community members](http://discord.vuestorefront.io/)
```

Comment:
```suggestion
- active [Discord community](http://discord.vuestorefront.io/)
```

Since our move to Discord, this number is much lower (getting close to 1000)